src/openapi.yaml

```diff
@@ -460,7 +460,6 @@ paths:
             type: array
             items:
               $ref: '#/components/schemas/ProtocolStatus'
-          example: wip is this a deprecated = true OR an example?
```

Comment:
In reality "deleted" protocols are just set to status DEPRECATED, but now I think we don't need to explain this in this doc - it's an implementation detail. So that part of the description can be removed.